src/DataStreams/ExecutionSpeedLimits.h

```diff
@@ -3,6 +3,7 @@
 #include <Poco/Timespan.h>
 #include <common/types.h>
 #include <DataStreams/SizeLimits.h>
+#include <Common/Stopwatch.h>
```

Comment:
BTW if only reference/pointer is used, forward declaration is enough.

Reply:
It's quite a light header, but sure: https://github.com/ClickHouse/ClickHouse/pull/27569